test/system/505-networking-pasta.bats

```diff
@@ -0,0 +1,1348 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Networking with pasta(1)
+#
+# Copyright (c) 2022 Red Hat GmbH
+# Author: Stefano Brivio <sbrivio@redhat.com>
+
+load helpers
+
+
+### Addresses ##################################################################
+
+@test "podman networking with pasta(1) - IPv4 default address assignment" {
+	skip_if_no_pasta
+	skip_if_no_ipv4
+
+	run_podman run --net=pasta $IMAGE ip -j -4 addr show
+
+	output="$(ipv4_get_addr_global "${output}")"
+	assert "$(ipv4_get_addr_global)"
```

Comment:
Please don't use the one-argument form. Use something like
```bash
   assert "$myvar" = "something" "descriptive name for when the test fails"

Reply:
I added a description for the test failure, using the two-argument form -- not the one with three arguments, though, as I really want to match on the output. I could make it a bit longer, but to be consistent here I would introduce a lot of cruft...